aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -473,10 +481,32 @@ Resources:
       NetworkInterfaces:
         - AssociatePublicIpAddress: true
           DeviceIndex: 0
-          GroupSet:
-            - !ImportValue VPC-FrontendSecurityGroup
-            - !ImportValue VPC-GatewaySecurityGroup
-          SubnetId: !ImportValue VPC-PublicSubnet<%=azs.first%>
+<%if stack_name != 'staging-next' -%>
+      GroupSet:
+        - !ImportValue VPC-FrontendSecurityGroup
+        - !ImportValue VPC-GatewaySecurityGroup
+<% else -%>
+          GroupSet: [Ref: WebServerSecurityGroup]
+  WebServerSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Enable HTTP[S] + SSH access
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: <%=ssh_ip%>
+      VpcId: !ImportValue VPC
+<% end -%>
```

Comment:
This is specifically to support a no-replacement migration (and can be removed once the `staging-next` stack is no longer in service). Since the `NetworkInterfaces` property on the `EC2::Instance` resource does not support in-place updates, we need to preserve the exact set of associated resources currently used by the `standalone` stack here, and manually adjust the resources afterwards.